libs/agno/agno/team/team.py

```diff
@@ -439,17 +444,17 @@ def _initialize_member(self, member: Union["Team", Agent], session_id: Optional[
             member.team_session_id = session_id
 
         # Set the team session state on members
-        if self.session_state is not None:
+        if self.team_session_state is not None:
             if isinstance(member, Agent):
```

Comment:
Now we don't need to make a distinction between Agent/Team, since they are the same. This can be simplified.